golem-worker-service/src/api/openapi/converter.rs

```diff
@@ -0,0 +1,1019 @@
+        }
+
+        paths
+    }
+
+    fn generate_operation(&self, route: &Route) -> Operation {
+        match &route.binding {
+            BindingType::Default { .. } => {
+                Operation {
+                    summary: Some(route.description.clone()),
+                    description: None,
+                    operation_id: Some(format!("{}_{}",
+                        route.template_name.to_snake_case(),
+                        route.method.to_string().to_lowercase())),
+                    parameters: {
+                        let mut params = Self::extract_path_parameters(&route.path).unwrap_or_default();
+                        params.extend(Self::extract_query_parameters(route));
+                        params.extend(Self::extract_header_parameters(route));
+                        if params.is_empty() { None } else { Some(params) }
+                    },
+                    request_body: Self::create_request_body(route),
+                    responses: {
+                        let mut map = Self::create_responses(route, "*");
+                        // Add CORS headers to all responses
+                        for response in map.values_mut() {
+                            response.headers = Some(Self::create_cors_headers("*"));
+                        }
+                        map
+                    },
+                    security: None,
+                    tags: Some(vec![route.template_name.clone()]),
+                }
+            },
+            BindingType::FileServer { .. } => {
+                Operation {
+                    summary: Some(route.description.clone()),
+                    description: None,
+                    operation_id: Some(format!("{}_{}",
+                        route.template_name.to_snake_case(),
+                        route.method.to_string().to_lowercase())),
+                    parameters: {
+                        let mut params = Self::extract_path_parameters(&route.path).unwrap_or_default();
+                        params.extend(Self::extract_query_parameters(route));
+                        params.extend(Self::extract_header_parameters(route));
+                        if params.is_empty() { None } else { Some(params) }
+                    },
+                    request_body: Self::create_request_body(route),
+                    responses: {
+                        let mut map = Self::create_responses(route, "*");
+                        // Add CORS headers to all responses
+                        for response in map.values_mut() {
+                            response.headers = Some(Self::create_cors_headers("*"));
+                        }
+                        map
+                    },
+                    security: None,
+                    tags: Some(vec![route.template_name.clone()]),
+                }
+            },
+            BindingType::SwaggerUI { .. } => {
+                Operation {
```

Comment:
For SwaggerUI, I would expect it to be documented as any endpoint which hosts a SwaggerUI (probably similar to the file server, but where the files being served are constrained to SwaggerUI files).